### ISSUE TITLE
trouble with ssh

### ISSUE THREAD

**Issue by ifuchs** [holzschu/a-shell#14]
I tried unsuccessfully to run ssh on a local host. It says "The authenticity of host xxx can't be established" and " Permanently added key for xxx to known hosts". But then it says "No keys found in directory .../.ssh." It has created a file named known_hosts but ther eis no dir called .ssh. I tried creating .ssh and moving the known_hosts file there but the result is the same. Does ssh work?

**Comment by holzschu** [comment]
Yes, it does work, but the authentification method is through keys, so you must have your private key in `~/Documents/.shh/` and the public key in the `~/.ssh/authorized_keys` of the host machine.

You can generate keys with `ssh-keygen`, or copy them.

**Comment by ryanbooker** [comment]
Generating keys on the iPad, seems to freeze as soon as you accept the name.

**Comment by holzschu** [comment]
Oh. I see that one. It will be fixed in the next version.
